command/server_util.go

```diff
@@ -58,7 +58,7 @@ func adjustCoreForSealMigration(logger log.Logger, core *vault.Core, barrierSeal
 		// In this case our migration seal is set so we are using it
 		// (potentially) for unwrapping. Set it on core for that purpose then
 		// exit.
-		core.SetSealsForMigration(nil, nil, unwrapSeal)
+		core.SetSealsForMigration(unwrapSeal, barrierSeal, unwrapSeal)
```

Comment:
@jefferai My auto->auto seal migration tests are failing without this change, but I don't understand why it was this way or what I might be breaking with it.  Do you have any insight to share?